client/src/components/History/Content/model/states.ts

```diff
@@ -103,6 +104,14 @@ export const STATES: StateMap = {
         text: "This dataset is being fetched.",
         icon: "spinner",
         spin: true,
+        nonDb: true,
+    },
+    /** the collection is empty. This state is only visual and transitional, it does not exist in the database. */
+    emptyCollection: {
+        status: "secondary",
+        text: "This is an empty list/collection.",
```

Comment:
For what you want to do this is also not correct. You want to highlight a collection that is not populated, that is not the same as being empty.